atmo/jobs/views.py

```diff
@@ -136,6 +136,24 @@ def detail_spark_job(request, id):
 
 @login_required
 @view_permission_required(SparkJob)
+@modified_date
+def detail_zeppelin_job(request, id):
+    """
+    View to show the details for the scheduled Zeppelin job with the given ID.
+    """
+    spark_job = SparkJob.objects.get(pk=id)
```

Comment:
Use `get_object_or_404` here. So if the ID doesn't exist or is invalid it will 404 instead of 500.